clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2033,6 +2033,31 @@ def MemCpyOp : CIR_Op<"libc.memcpy"> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// FAbsOp
+//===----------------------------------------------------------------------===//
+
+def FAbsOp : CIR_Op<"libc.fabs", [Pure, SameOperandsAndResultType]> {
```

Comment:
Since this usually mapped to intrinsic at some point, we can probably drop the `libc` prefix.